mlflow/__init__.py

```diff
@@ -43,6 +43,7 @@
 import mlflow.projects as projects
 import mlflow.tracking as tracking
 import mlflow.models
+import mlflow.artifacts
```

Comment:
Include `mlflow.artifacts` in the init file so that users can call its functions simply by importing `mlflow` first (no need to import `mlflow.artifacts` explicitly)